cmd/api-errors.go

```diff
@@ -800,6 +801,12 @@ var errorCodes = errorCodeMap{
 		Description:    "x-amz-object-lock-retain-until-date and x-amz-object-lock-mode must both be supplied",
 		HTTPStatusCode: http.StatusBadRequest,
 	},
+	ErrOBDLockTimeout: {
+		Code:           "UnsupportedOperation",
+		Description:    "parallel OBD requests are not supported at this time",
+		HTTPStatusCode: http.StatusConflict,
```

Comment:
We can return `http.StatusRequestTimeout` since the error is Timeout? if not rename it as conflict?

Reply:
The error is not a timeout. It is an error sent back to clients that try to start a new OBD task while one is already running

Reply:
Then we should say `ErrOBDLockTimeout--> ErrAdminOBDConflict`